verl/trainer/ppo/ray_trainer.py

```diff
@@ -775,6 +634,12 @@ def init_workers(self):
                 config=self.config, worker_group=self.actor_rollout_wg, rm_wg=self.rm_wg
             )
 
+        # A CPU workgroup is created to take over the computational tasks
+        # that drive the process, such as advantage computing.
+        resource_pool = RayResourcePool([1], use_gpu=False, name_prefix="cpu_pool")
```

Comment:
Can we dispatch computation, e.g `compute_advantage`, `compute_rollout_importance_weights_and_add_to_batch` to actor worker group? For example, we may need GPU to accelerate `compute_advantage`.